autoprecompiles/src/lib.rs

```diff
@@ -95,12 +93,6 @@ impl<T: Clone + Ord + std::fmt::Display> SymbolicBusInteraction<T> {
     }
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize, Eq, PartialEq, Hash)]
-pub enum BusInteractionKind {
-    Send,
-    Receive,
-}
-
```

Comment:
I think we should still have this here otherwise users of this crate will need to depend on `ast` too

Reply:
I assume you meant `powdr-openvm`. I leave both and do the transformation here in `autoprecompiles`, this way `powdr-openvm` doesn't need the one in `ast` (and I avoid having to import `BusInteractionKind` from `autoprecompiles` to `ast` which is also weird).